client_taxi_driver/shared/src/commonMain/kotlin/domain/usecase/ManageOrderUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package domain.usecase
+
+import data.remote.fakegateway.MapFakeGateway
+import domain.entity.Order
+import kotlinx.coroutines.delay
+
+
+interface IManageOrderUseCase {
+    suspend fun foundNewOrder(): Order
+}
+
+class ManageOrderUseCase(
+    private val fakeGateway: MapFakeGateway,
+) : IManageOrderUseCase {
+    override suspend fun foundNewOrder(): Order {
+        delay(3000)
```

Comment:
if this for the fake gateway it's wrong to be here move it to the fake gateway

Reply:
done